lib/web_ui/lib/pointer.dart

```diff
@@ -153,7 +153,23 @@ class PointerData {
   }
 }
 
+/// A function used to notify the engine about how a [PointerDataPacket] was used in the framework.
+///
+/// For example, can be used to [preventDefault] the source events on the web.
+typedef _PointerDataPacketAcknowledgeFn = void Function({
+  bool? preventDefault,
+});
+
+// The default _PointerDataPacketAcknowledgeFn, which is a noop.
+void _defaultPointerDataPacketAcknowledgeFn ({
+  bool? preventDefault,
+}) {}
+
 class PointerDataPacket {
-  const PointerDataPacket({this.data = const <PointerData>[]});
+  const PointerDataPacket({
+    this.data = const <PointerData>[],
+    this.acknowledge = _defaultPointerDataPacketAcknowledgeFn,
+  });
   final List<PointerData> data;
+  final _PointerDataPacketAcknowledgeFn acknowledge;
```

Comment:
acknowledge is kind of an awkward name for this. Makes it sound like it has to be called for every package?

Reply:
I'm trying to not use the best names for when this gets re-done. Definitely don't want to make it sound like a mandatory call (it's the opposite :P) What about `answer`?